### ISSUE TITLE
SyntaxError: Unexpected token import

### ISSUE THREAD

**Issue by yshrsmz** [DavidBernal/nuxt-property-decorator#7]
I got following error in v1.1.3.
when I use v1.0.0, it works fine.
seems like there's no `nuxt-property-decorator.umd.js` in v1.1.3, and entry point of the library is `nuxt-property-decorator.js` which uses `import` (and I believe node.js does not support `import` syntax out of the box yet)

```
{ /Users/foo/repos/example/node_modules/nuxt-property-decorator/lib/nuxt-property-decorator.js:2
import Vue from 'vue';
^^^^^^

SyntaxError: Unexpected token import
    at new Script (vm.js:51:7)
    at createScript (vm.js:138:10)
    at Object.runInThisContext (vm.js:199:10)
    at Module._compile (module.js:624:28)
    at Object.Module._extensions..js (module.js:671:10)
    at Module.load (module.js:573:32)
    at tryModuleLoad (module.js:513:12)
    at Function.Module._load (module.js:505:3)
    at Module.require (module.js:604:17)
    at require (internal/module.js:11:18)
    at r (/Users/yshrsmz/repos/androiddagashi/node_modules/vue-server-renderer/build.js:8330:16)
    at Object.<anonymous> (server-bundle.js:20887:18)
    at __webpack_require__ (webpack:/webpack/bootstrap 9be7dd6ad8a646157908:25:0)
    at Object.70 (pages/index.vue?c13f:1:0)
    at __webpack_require__ (webpack:/webpack/bootstrap 9be7dd6ad8a646157908:25:0)
    at Object.66 (pages/index.vue:1:0)
    at __webpack_require__ (webpack:/webpack/bootstrap 9be7dd6ad8a646157908:25:0)
    at <anonymous>
    at process._tickCallback (internal/process/next_tick.js:160:7) statusCode: 500, name: 'SyntaxError' }
```

**Comment by meenie** [comment]
Getting the same.

**Comment by DavidBernal** [comment]
Fixed in 1.1.4. Touch me if the error persist.

**Comment by meenie** [comment]
Hi @DavidBernal, it's still broken but with a different error.  The file you are pointing to in the `main` property of the `package.json` is not present when downing your library from NPM.

![image](https://user-images.githubusercontent.com/1126819/37876781-7669a8c0-3006-11e8-8d4f-fa99ae92c29b.png)
